docs/resources/aws_ec2_transit_gateway_route_table_associations.md

```diff
@@ -0,0 +1,79 @@
+---
+title: About the aws_ec2_transit_gateway_route_table_associations Resource
+platform: aws
+---
+
+# aws\_ec2\_transit\_gateway\_route_table\_associations
+
+Use the `aws_ec2_transit_gateway_route_table_associations` InSpec audit resource to test properties of some or all AWS transit gateway route table associations.
+
+An AWS transit gateway route table association associates the specified attachment with the specified transit gateway route table. You can associate only one route table with an attachment.
+
+## Syntax
+
+Verify that a transit gateway route table ID exists.
+
+    describe aws_ec2_transit_gateway_route_table_associations(transit_gateway_route_table_id: 'TRANSIT_GATEWAY_ROUTE_TABLE_ID') do
+      it { should exist }
+    end
+
+## Parameters
+
+`transit_gateway_route_table_id` _(required)_
+
+See the [AWS documentation on transit gateway route table](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-transitgatewayroutetableassociation.html) for additional information.
```

Comment:
<!--StartFragment-->For additional information, check out the [AWS documentation on transit gateway route table](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-transitgatewayroutetableassociation.html) documentation.<br class="Apple-interchange-newline"><!--EndFragment-->